### PR TITLE
351 trajectory or row

### PR DESCRIPTION
**Pull request by kevinsantana11**
Hey @philippemiron Phelippe I went ahead and created a separate branch from yours. From there I merged Cloud-Drift/main into it and made some changes on top of that to get our tests working. I also tested the ragged array data structure creation from the gdp1h adapter for 10 drifters and everything looked good to me. 

Please review the changes as there were a good bit of conflicts and I want to make sure I didn't accidentally smudge one of your changes.

### CODE REVIEW

**Comment by kevinsantana11** [comment]
@philippemiron

File: clouddrift/adapters/gdp6h.py [hunk 1]
```diff
@@ -513,9 +509,9 @@ def to_raggedarray(
         indices=ids,
         preprocess_func=preprocess,
         name_coords=gdp.GDP_COORDS,
-        name_dims=gdp.GDP_DIMS,
         name_meta=gdp.GDP_METADATA,
         name_data=GDP_DATA,
+        dim_names=gdp.GDP_DIMS,
```
Comment (philippemiron) [inline clouddrift/adapters/gdp6h.py]: what do you think of `name_dims` instead so it matches with `name_coords`, `name_meta`, and `name_data`.

**Comment by philippemiron** [comment]
I'm still not sure how this is going to work? Won't we still have to resolve the merge conflict when going from my branch to `clouddrift/main`? You could have committed directly on my branch.

File: clouddrift/adapters/gdp6h.py (hunk 1 above)
Comment (kevinsantana11) [reply clouddrift/adapters/gdp6h.py]: that makes sense, I could also swap the mapping as it should reduce the complexity of the app

**Comment by kevinsantana11** [comment]
> I'm still not sure how this is going to work? Won't we still have to resolve the merge conflict when going from my branch to `clouddrift/main`? You could have committed directly on my branch.

Once me merge this into your branch it will be the same as committing on your branch directly. I just wanted to be careful here and have another review before making the big change.

**Comment by kevinsantana11** [comment]
@philippemiron updated to reflect the change of using name_dims over dim_names for consistency

**Pull request merged**